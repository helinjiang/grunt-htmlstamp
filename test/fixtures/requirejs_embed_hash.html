<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>requirejs_embed_hash.html</title>
    <script data-main="./requirejs/page/requirejs_embed_hash.js" src="./requirejs/lib/require.js"></script>
</head>
<body>
<h1>requirejs_embed_hash.html</h1>
</body>
</html>