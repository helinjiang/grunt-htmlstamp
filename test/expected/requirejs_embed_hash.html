<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>requirejs_embed_hash.html</title>
    <script data-main="./requirejs/page/requirejs_embed_hash.47060b2018.js" src="./requirejs/lib/require.js"></script>
</head>
<body>
<h1>requirejs_embed_hash.html</h1>
</body>
</html>